reviews/templates/mail/submitted_longroute_other_applicants.html

```diff
@@ -0,0 +1,35 @@
+{% extends "mail/base.html" %}
+
+{% block content_nl %}
+{% if was_revised %}
+Beste collega,<br/>
+<br/>
+Uw collega {{ creator }} heeft een gereviseerde aanvraag ingediend voor een onderzoek, waar u aan meewerkt, bij de ethische commissie. De aanvraag is door de FETC-GW ontvangen.<br/>
```

Comment:
Which application? Is what I would ask when reading this sentence. 

As the recipient might not  be aware they were added as an applicant, or if the recipient is added to a lot of applications, it's not immediately obvious what the email is talking about. 

It would be nice if the title of the proposal is given as well. 

(Same comment applies to the other mails as well)